### PR TITLE
LPS-197235 [MARKETPLACE] [BUG] Remove phone number validation from Clarity Solution Page

### PR DESCRIPTION
**Pull request by joanagrego**

### CODE REVIEW

**Comment by joanagrego** [comment]
ci:test:sf

**Comment by joanagrego** [comment]
ci:test:sf

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedSolutions/PurchasedSolutions.tsx [hunk 1]
```diff
@@ -551,15 +554,22 @@ const PurchasedSolutions: React.FC = () => {
...
+														>
+															Terms & Conditions
+														</ClayLink>
+													</label>
+													<label
+														className="ml-4"
+														htmlFor="agreeToTermsAndConditions"
+													>
+														I agree to the
+													</label>
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedSolutions/PurchasedSolutions.tsx:571]:
```suggestion
                                                                                                         <label
														className="ml-2"
														htmlFor="agreeToTermsAndConditions"
													>
														I agree to the
													</label>
													<label className="ml-4">
														<ClayLink
															displayType="primary"
															href="https://www.liferay.com/en/legal/marketplace-terms-of-service"
														>
															Terms & Conditions
														</ClayLink>
													</label>
													
```

Comment (kevenleone) [reply workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedSolutions/PurchasedSolutions.tsx:571]: No reverse 😄

**Review by kevenleone — CHANGES_REQUESTED** [review]

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedSolutions/PurchasedSolutions.tsx [hunk 2]
```diff
@@ -327,6 +328,8 @@ const PurchasedSolutions: React.FC = () => {
 
 	const agreeToTermsAndConditions = watch('agreeToTermsAndConditions');
 
+	const hasAllValidations = isValid && agreeToTermsAndConditions;
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedSolutions/PurchasedSolutions.tsx:331]:
```suggestion
	const hasAllValidations = agreeToTermsAndConditions  && isValid;
```

**Review by kevenleone — CHANGES_REQUESTED** [review]

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedSolutions/PurchasedSolutions.tsx (hunk 1 above)
Comment (joanagrego) [reply workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedSolutions/PurchasedSolutions.tsx:571]: Done!

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedSolutions/PurchasedSolutions.tsx (hunk 2 above)
Comment (joanagrego) [reply workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedSolutions/PurchasedSolutions.tsx:331]: Done.

**Comment by joanagrego** [comment]
ci:test:sf

**Comment by joanagrego** [comment]
liferay/ci:test:site-initializer-liferay-marketplace

**Comment by joanagrego** [comment]
ci:test:site-initializer-liferay-marketplace

**Review by kevenleone — APPROVED** [review]
LGTM 🔥

**Comment by kevenleone** [comment]
ci:test:relevant

**Comment by LuanaKSilva** [comment]
Hey @joanagrego, I analyzed the errors and none of them are caused because of your PR. Feel free to go ahead.

**Comment by joanagrego** [comment]
Resent --> [PR](https://github.com/brianchandotcom/liferay-portal/pull/141494)

**Pull request closed without merging**